packages/office-ui-fabric-react/src/components/Callout/docs/CalloutOverview.md

```diff
@@ -1,5 +1,7 @@
 Callouts are a powerful way to simplify a user interface. They host tips and other information users need when they need it, with minimal effort on their part. Callouts can help you use screen space more effectively and reduce screen clutter. However, poorly designed Callouts can be annoying, distracting, unhelpful, overwhelming, or in the way.
 
+Callouts that do not contain focusable elements (links, buttons etc) cannot gain focus using `setInitialFocus`. For proper screen reader support follow the `Non Focusable Callout Example` which treats the callout content like a status message.
+
 Use a Callout for displaying additional contextual information about an item on the screen. Callouts also have a tail that identifies their source. A common use for Callout is the introduction of a new feature or capability of an app or site. Alternate usages include pairing the Callout with a button or clickable element for on-demand presentation of additional or supporting content.
 
-Real-world examples of this implementation can be seen in administrative interfaces where a particularly difficult-to-understand concept is paired with the iconClassNames.info "i" icon. In this example, Callout - with its tip text - is opened when the user clicks on or hovers over the icon.
\ No newline at end of file
+Real-world examples of this implementation can be seen in administrative interfaces where a particularly difficult-to-understand concept is paired with the iconClassNames.info "i" icon. In this example, Callout - with its tip text - is opened when the user clicks on or hovers over the icon.
```

Comment:
I know you didn't add it but this paragraph is unnecessarily complex and specific. I'd suggest something like this instead (also we ideally ought to have an example showing this pattern)...don't totally love this but I think it's an improvement.

```suggestion
One example of Callout usage is when an "i" icon is displayed in the label of a form element which might require additional explanation. A Callout is opened to display tip text when the user selects or hovers over the icon.
```

Reply:
Umm actually we should remove this paragraph...this is TooltipHost behavior, not something we should mention in the Callout article.